wsd/LOOLWSD.cpp

```diff
@@ -2794,15 +2780,7 @@ class ClientRequestDispatcher : public SimpleSocketHandler
                 errMsg = "Empty clipboard item / session tag " + tag;
 
             // Bad request.
-            std::ostringstream oss;
-            oss << "HTTP/1.1 400\r\n"
-                << "Date: " << Util::getHttpTimeNow() << "\r\n"
-                << "User-Agent: LOOLWSD WOPI Agent\r\n"
-                << "Content-Length: 0\r\n"
-                << "\r\n"
-                << errMsg;
-            socket->send(oss.str());
-            socket->shutdown();
+            socket->sendError(400, errMsg, "", true);
```

Comment:
We should have named enum values for all the HTTP error codes. Having 400 hard-coded is a very error prone, hard to remember what it means, etc. Best extract them into an enum and give them standard HTTP names.

Reply:
Hmm, generally I'd agree. But does it make sense to define a custom enum for that? Something like this should already exist, so we could use that, right?